requirements_dev.txt

```diff
@@ -10,4 +10,4 @@ Sphinx==1.3.1
 PyYAML==3.11
 pytest==2.8.7
 prospector==0.11.7
-
+robotframework-lint==0.7
```

Comment:
i havent generally been putting the linters in the dev requirements but maybe thats not a bad idea